From 9102bcf74c6b3e1cd02aac2c4b1dd9c14f7e1d07 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Mon, 23 Dec 2013 19:57:56 +0800
Subject: [PATCH] Fix the check for lex program.

Flex is required for processing src/css.l.  AC_PROG_LEX defaults to
`lex` or `:' when `flex' cannot be found.

Signed-off-by: Yousong Zhou <yszhou4tech@gmail.com>
---
 configure.ac |    8 +++++++-
 1 files changed, 7 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index b24588f..0a9ec59 100644
--- a/configure.ac
+++ b/configure.ac
@@ -112,7 +112,13 @@ AM_GNU_GETTEXT_VERSION([0.17])
 
 AC_PROG_RANLIB
 
-AC_PROG_LEX
+dnl
+dnl Flex
+dnl
+AC_CHECK_PROGS([LEX], [flex], [:])
+if test "$LEX" = :; then
+  AC_MSG_ERROR([No available flex program was found.])
+fi
 
 dnl Turn on optimization by default.  Specifically:
 dnl
-- 
1.7.2.5

